exporter/azuremonitorexporter/logexporter.go

```diff
@@ -39,6 +40,21 @@ func (exporter *logExporter) onLogData(_ context.Context, logData plog.Logs) err
 	return nil
 }
 
+func (exporter *logExporter) Shutdown(_ context.Context) error {
+	shutdownTimeout := 1 * time.Second
+
+	select {
+	case <-exporter.transportChannel.Close():
+		return nil
+	case <-time.After(shutdownTimeout):
+		// Currently, due to a dependency's bug (https://github.com/microsoft/ApplicationInsights-Go/issues/70),
```

Comment:
Let me know if you disagree with my reasoning here, happy to discuss further.

`This is not an error` might be a bit misleading as technically we want the channel `Close` returns to send a message before the timeout is hit, but I think it captures the idea.